src/Components/Layout/ResponsiveGrid.tsx

```diff
@@ -19,9 +22,9 @@ export default function ResponsiveGrid() {
         spacing={{ xs: 2, md: 3 }}
         columns={{ xs: 12, sm: 12, md: 12 }}
       >
-        {Array.from(Array(12)).map((_, index) => (
+        {Array.from(Array(4)).map((_, index) => (
```

Comment:
Maybe a const named value which will be store in Consts instead of 4 / 12? Will be more readable what it is